delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/acls/Acl.scala

```diff
@@ -113,4 +128,13 @@ object Acl {
 
   implicit def aclJsonLdEncoder(implicit base: BaseUri): JsonLdEncoder[Acl] =
     JsonLdEncoder.fromCirce(context)
+
+  implicit private[Acl] val config: Configuration = Configuration.default.copy(transformMemberNames = {
+    case "path" => nxv.path.prefix
+    case other  => other
+  })
+
+  implicit val metadataEncoder: Encoder.AsObject[Metadata] = deriveConfiguredEncoder[Metadata]
```

Comment:
I think this one should be private and only the `JsonLdEncoder[Metadata]` public